### ISSUE TITLE
Broken on Sopel master (7.0.0-dev)

### ISSUE THREAD

**Issue by dgw** [sopel-irc/sopel-github#48]
After updating my own Sopel bot to `master`, the GitHub plugin stopped loading. It seems related to the webhook functionality:

```
[2019-10-31 18:45:50,344] sopel.modules.reload ERROR    - Error loading github: 'Connection' object has no attribute 'cursor'
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/sopel/modules/reload.py", line 26, in _load
    plugin.setup(bot)
  File "/usr/local/lib/python3.5/dist-packages/sopel/plugins/handlers.py", line 252, in setup
    self._module.setup(bot)
  File "/usr/local/lib/python3.5/dist-packages/sopel_modules/github/github.py", line 96, in setup
    setup_webhook(sopel)
  File "/usr/local/lib/python3.5/dist-packages/sopel_modules/github/webhook.py", line 50, in setup_webhook
    c = conn.cursor()
AttributeError: 'Connection' object has no attribute 'cursor'
```

Since the custom DB stuff in this plugin was written for when Sopel used SQLite directly, and it now uses SQLAlchemy to abstract the database away, I'd say that's what broke this.

I'm doubtful that we can fix this on the Sopel side, so I'm opening the issue here. SQLAlchemy just _won't_ be the same as using `sqlite3.connect()` because, well, it's different. @RustyBower your input/help patching would be much appreciated, especially if you can tell me that this _is_ fixable on the Sopel side so we can keep old code working! 😉

(Please don't judge me for py3.5; it's the newest I could install without compiling my own. 😅)

**Comment by RustyBower** [comment]
Did a quick pass on this, but will need to set up a dev env with a webhook and such to get this fully tested.

**Comment by dgw** [comment]
This is fixed with sopel-irc/sopel#1811
